packages/components/src/vue/components/morph-parts/definitions-list.vue

```diff
@@ -44,7 +44,12 @@ export default {
       if (this.$store.getters['app/shortDefDataReady'] && this.lexeme && this.lexeme.meaning && this.lexeme.meaning.shortDefs && this.lexeme.meaning.shortDefs.length > 0) {
         definitionsLocal = this.lexeme.meaning.shortDefs
       } else if (this.lexeme && this.lexeme.lemma.features && Object.entries(this.lexeme.lemma.features).length > 0) {
-        definitionsLocal = [new Definition(this.l10n.getMsg('TEXT_NOTICE_NO_DEFS_FOUND'), 'en-US', 'text/plain', this.lexeme.lemma.word)]
+        definitionsLocal = [
+          new Definition(
+            this.l10n.getMsg('TEXT_NOTICE_NO_DEFS_FOUND'), Language.ENGLISH,
```

Comment:
@kirlat , some comments were erased somehow - I would repeat with this ine - because it is a error.
You pass `Language.English` instead of 'en-US' String, I believe `getMsg` method does not support passing Object.
You should fix this, I think.